### ISSUE TITLE
Magisk stays on splash screen after hiding App.

### ISSUE THREAD

**Issue by Whosondephone** [topjohnwu/Magisk#8112]
I just updated from 20.4. Before the update I restored the Hidden app back to normal. Installed Magisk 27 APP. And installed Magisk directly within the APP.  I then attempted to hide again.

I was staying on 20.4 because I was having a similar issue loading the APP.

Device: LG V20
Android version: 8.0
Magisk version name: 27002 D
Magisk version code: 27002

Logcat legitimately scares me.
[Magisk270002-05_06-00-48-15_532.zip](https://github.com/user-attachments/files/15573473/Magisk270002-05_06-00-48-15_532.zip)
[Magisk270002-05_06-00-48-15_532.log](https://github.com/user-attachments/files/15573475/Magisk270002-05_06-00-48-15_532.log)

**Comment by canyie** [comment]
Looks like caused by another bug in LG framework implementation.
```
1717562969.636  root 23505 23505 E IPC     : Error in IPCMain
1717562969.636  root 23505 23505 E IPC     : java.lang.reflect.InvocationTargetException
1717562969.636  root 23505 23505 E IPC     : 	at java.lang.reflect.Method.invoke(Native Method)
1717562969.636  root 23505 23505 E IPC     : 	at com.topjohnwu.superuser.internal.RootServerMain.<init>(SourceFile:0)
1717562969.636  root 23505 23505 E IPC     : 	at com.topjohnwu.superuser.internal.RootServerMain.main(Unknown Source:23)
1717562969.636  root 23505 23505 E IPC     : 	at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
1717562969.636  root 23505 23505 E IPC     : 	at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:284)
1717562969.636  root 23505 23505 E IPC     : Caused by: java.lang.ExceptionInInitializerError
1717562969.636  root 23505 23505 E IPC     : 	at com.lge.systemservice.core.LGContext.<clinit>(LGContext.java:419)
1717562969.636  root 23505 23505 E IPC     : 	at android.app.ApplicationPackageManager.getLGPackageLabelService(ApplicationPackageManager.java:1667)
1717562969.636  root 23505 23505 E IPC     : 	at android.app.ApplicationPackageManager.<init>(ApplicationPackageManager.java:1477)
1717562969.636  root 23505 23505 E IPC     : 	at android.app.ContextImpl.getPackageManager(ContextImpl.java:235)
1717562969.636  root 23505 23505 E IPC     : 	at android.content.ContextWrapper.getPackageManager(ContextWrapper.java:94)
1717562969.636  root 23505 23505 E IPC     : 	at com.topjohnwu.magisk.DelegateRootService.attachBaseContext(DelegateRootService.java:28)
1717562969.636  root 23505 23505 E IPC     : 	... 5 more
1717562969.636  root 23505 23505 E IPC     : Caused by: android.content.res.Resources$NotFoundException: Resource ID #0x60044
1717562969.636  root 23505 23505 E IPC     : 	at android.content.res.ResourcesImpl.getValue(ResourcesImpl.java:223)
1717562969.636  root 23505 23505 E IPC     : 	at android.content.res.Resources.getBoolean(Resources.java:1065)
1717562969.636  root 23505 23505 E IPC     : 	at com.lge.systemservice.core.LGMiniClockManager.<clinit>(LGMiniClockManager.java:23)
1717562969.636  root 23505 23505 E IPC     : 	... 11 more
1717562969.639  root 23505 23505 I g.qnllyy:root:0: System.exit called, status: 1
1717562969.639  root 23505 23505 I AndroidRuntime: VM exiting with result code 1.
```

**Comment by pndwal** [comment]
Also, seems latest stock is 9.0 Pie... Would that help?

**Comment by Whosondephone** [comment]
Only the Korean variant got Pie.

**Comment by yujincheng08** [comment]
Should be fixed by cfd1e0cf2266919b91b18c367263885f6fec3322

**Comment by Whosondephone** [comment]
App version 27003 D working perfectly thank you!
